apps/modify/modify.go

```diff
@@ -0,0 +1,100 @@
+package modify
```

Comment:
This package needs a package comment (and can we put it under `.../confgenerator/fluentbit/modify` since it appears to be helpers for generating fluentbit config?)